src/ContinuosBondingERC20Token.sol

```diff
@@ -114,6 +123,9 @@ contract ContinuosBondingERC20Token is IContinuousBondingERC20Token, ERC20, Reen
         if (tokensToReceive < minExpectedAmount) revert InSufficientAmountReceived();
 
         _transfer(address(this), recipient, tokensToReceive);
+        if (isAntiWhaleFlagEnabled && block.timestamp - creationTime < ANTI_WHALE_TIME_PERIOD && balanceOf(recipient) > ANTI_WHALE_MAX_TOKEN) {
```

Comment:
```suggestion
        if (isAntiWhaleFlagEnabled && block.timestamp - creationTime < ANTI_WHALE_TIME_PERIOD && balanceOf(recipient) >= ANTI_WHALE_MAX_TOKEN) {
```

Reply:
Ohk yeah! I missed the fact that this line is after the _transfer function (which updates the balance before this check).. All good, thanks for clarification.